src/netdef/NetdefToNet.cpp

```diff
@@ -271,4 +265,4 @@ STATIC bool NetdefToNet::createNetFromNetdef(NeuralNet *net, std::string netdef,
     }
     net->addLayer(SoftMaxMaker::instance());
     return true;
-}
+}
```

Comment:
prefer to have a newline at the end of the file on the whole.  This is consistent eg with `pep8`.  also, see https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning